### DISCUSSION TITLE
编辑md后右上角有未同步，等下后变成已同步，这是同步了啥

### DISCUSSION THREAD

**Discussion by goodlucktogzx** [shuzijun/markdown-editor#109] [Q&A]
![image](https://github.com/shuzijun/markdown-editor/assets/48192962/67147258-10c5-44d7-9031-21944021bde4)

**Comment by goodlucktogzx** [comment]
![image](https://github.com/shuzijun/markdown-editor/assets/48192962/5551ff9b-5861-49f6-9cc1-e1e65e813342)

**Comment by goodlucktogzx** [comment]
![image](https://github.com/shuzijun/markdown-editor/assets/48192962/9ed74fe7-f29e-4f3d-950d-6d6a19479d48)

**Comment by shuzijun** [comment]
编辑后是存在了编辑内嵌的网页里，同步是指将编辑器网页的内容保存到本地文件里,可以理解为执行一次保存操作.
